app/routes.py

```diff
@@ -37,6 +37,8 @@
 newest_version = '' if current_version >= newest_version \
     else newest_version
 
+ac_var = 'WHOOGLE_AUTOCOMPLETE'
+autocomplete_enabled = os.getenv(ac_var, '1')
```

Comment:
pep8 also enforces 2 line breaks before function defs

```suggestion
autocomplete_enabled = os.getenv(ac_var, '1')

```